e2e/__tests__/readInitialOptions.test.ts

```diff
@@ -64,9 +64,21 @@ describe('readInitialOptions', () => {
     expect(config).toEqual({jestConfig: 'package.json', rootDir});
     expect(configPath).toEqual(configFile);
   });
-  test('should read a jest.config.ts file', async () => {
```

Comment:
Yeah, that's what I thought, but wasn't sure if it would be desired to do. I've replaced all the base testcases with a single `test.each` in 66e69a1d5d59fdc3e32d0a148a23b2a5672283d4, and could go further with auto-generating the table via `fs.readdir`, but not sure if that's your preference or not.

Reply:
this seems good to me, thanks! 👍